### PR TITLE
[Telegram Action] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: bundles/action/org.openhab.action.telegram/README.md [hunk 1]
```diff
@@ -6,24 +6,23 @@ The Telegram Action service allows sending formatted messages to Telegram client
...
 The action can be configured in `services/telegram.cfg`.
 
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| bots | | Yes | Comma-separated list of `<bot-name>`s |
-| `<bot name>.chatId` | | Yes | chat id |
-| `<bot name>.token` | | Yes | authentication token |
-
+| Property            | Default | Required | Description                           |
+|---------------------|---------|-:-:------|---------------------------------------|
```
Comment (9037568) [inline bundles/action/org.openhab.action.telegram/README.md]: This is broken.

Comment (bgilmer77) [reply bundles/action/org.openhab.action.telegram/README.md]: @9037568 I believe my latest commit fixes the table.  Please look, and if it is okay, please merge.

Comment (9037568) [reply bundles/action/org.openhab.action.telegram/README.md]: Unfortunately, the latest commit does not make any changes to the table.

**Comment by 9037568** [comment]
Thanks, @bgilmer77 !

**Pull request merged**